rpc/client/rpc_test.go

```diff
@@ -294,3 +295,43 @@ func TestTx(t *testing.T) {
 		}
 	}
 }
+
+func TestTxSearch(t *testing.T) {
```

Comment:
Sweet. We should probably modify the dummy to return a tag so we can test that here too!